dut/drivers.wake

```diff
@@ -0,0 +1,188 @@
+
```

Comment:
I think we can split this file into two separate files:

1. A file that only deals with slurping up drivers that have already been created and committed to source control and with generating configuration-specific header files.
2. A file that only deals with automatically generating driver files from the DUH document.

The two steps are unrelated to each other, and there's no need for them to share any code. The latter shouldn't be dependent on a number of other things, including the object model. I want to submit a PR against this branch demonstrating how this could work.